app/scss/indexPage.scss

```diff
@@ -27,7 +28,7 @@
       }
 
       .warning {
-        color : #ff0000;
```

Comment:
hi. Why inside .warning are two times color?

Reply:
i don't think there is. this is showing the one we removed.